protocol/x/revshare/keeper/revshare.go

```diff
@@ -175,6 +175,11 @@ func (k Keeper) GetAllRevShares(
 	makerFees := fill.MakerFeeQuoteQuantums
 	netFees := big.NewInt(0).Add(takerFees, makerFees)
 
+	// net fees is 0 in case of liquidations where maker and taker fees are 0
+	if netFees.Sign() == 0 {
+		return types.RevSharesForFill{}, nil
+	}
+
 	affiliateRevShares, affiliateFeesShared, err := k.getAffiliateRevShares(ctx, fill, affiliatesWhitelistMap)
 	if err != nil {
 		return types.RevSharesForFill{}, err
```

Comment:
Comment on this [line](https://github.com/dydxprotocol/v4-chain/pull/2413/files#diff-1203e511f1b1da0901525ce89b0abb29818a2cc88b7f5e916f0734df1a7d9de6R188) below: error message `affiliate fees shared exceeds net fees"` is not accurate since we return error when `netFeesSubAffiliateFeesShared = 0` as well